### DISCUSSION TITLE
How do I disable rag?

### DISCUSSION THREAD

**Discussion by NasonZ** [nicobrauchtgit/obsidian-Smart2Brain#154] [Q&A] [answered]
I want to use my llm to help me clean up some  notes.

I want to ask my llm to renumber some lists, format some text.

The issue is that smart second brain  is stuck reducing notes. I don't need RAG for this task so how do i disable it?

![image](https://github.com/user-attachments/assets/a4755d92-a0a6-4b29-b34c-338bceab179b)
 etc.

**Comment by Leo310** [comment, marked as answer]
If you want to disable RAG, press the octopus icon at the bottom.

**Reply by NasonZ** [reply]
Thanks.

Is it possible to adjust/remove the system prompt for this mode?



![image](https://github.com/user-attachments/assets/ed033048-7de6-4c66-8678-c3204982004c)
